spec/factories/info_requests.rb

```diff
@@ -6,34 +6,34 @@
     public_body
     user
 
-    after_create do |info_request, evaluator|
-      FactoryGirl.create(:initial_request, :info_request => info_request)
+    after(:create) do |info_request, evaluator|
+      create(:initial_request, :info_request_id => info_request.id)
```

Comment:
I don't "care" either way as such – whichever works and doesn't cause a breakage/warning :)